src/main/java/Problem3/Movie.java

```diff
@@ -8,16 +8,27 @@
     UUID id;
 
     public Movie(String rating, String title) {
-        this.id = UUID.randomUUID();
+        id = UUID.randomUUID();
```

Comment:
please do not change given code. ask instructor first if you believe you have a strong reason to do it.